services/metadata/backend/data/societesavoie.json

```diff
@@ -0,0 +1,338 @@
+            {
+                "en": "Prosopography"
+            },
+            {
+                "en": "States of Savoy"
+            },
+            {
+                "en": "Women"
+            }
+        ],
+        "name": "Société de cour, élites et finances : de l’Hôtel des princes de Savoie à l’État moderne",
+        "shortcode": "0121",
+        "spatialCoverage": [
+            {
+                "__type": "URL",
+                "type": "Geonames",
+                "url": "https://www.geonames.org/3175395/",
+                "text": "Italy"
+            },
+            {
+                "__type": "URL",
+                "type": "Geonames",
+                "url": "https://www.geonames.org/2658434/",
+                "text": "Switzerland"
+            },
+            {
+                "__type": "URL",
+                "type": "Geonames",
+                "url": "https://www.geonames.org/3017382/",
+                "text": "France"
+            }
+        ],
+        "startDate": "2020-08-01",
+        "temporalCoverage": [
+            {
+                "de": "1269-1465",
+                "en": "1269-1465",
+                "fr": "1269-1465"
+            }
+        ],
+        "url": {
+            "__type": "URL",
+            "type": "URL",
+            "url": "https://admin.dasch.swiss/beta/project/0121",
+            "text": "Project Data on DSP"
+        }
+    },
+    "datasets": [
+        {
+            "__id": "http://ns.dasch.swiss/repository#dsp-0121-dataset-000",
+            "__type": "Dataset",
+            "__createdAt": "1676912018756953000",
+            "__createdBy": "dsp-metadata-gui",
+            "abstracts": [
+                {
+                    "en": "The ontology of the project includes 18 classes and 157 properties, describing the information collected on the individuals belonging to or surrounding the princely households of the court of Savoy (names, gender, functions, time of service, office, income...), as well as the family relationships that may exist between these persons. This information is drawn from accounting, normative and notarial sources, whose archival classifications are always indicated, so that the documents can be traced back.  Finally, bibliographic data can be added to provide additional details on persons and functions.     L’ontologie du projet comprend 18 classes et 157 propriétés, décrivant les informations récoltées sur les individus faisant partie des hôtels princiers de la cour de Savoie ou de leur entourage (noms, genre, fonctions, temps de service, office, revenus...), ainsi que les relations familiales qui peuvent exister entre ces personnes.  Ces informations sont tirées de sources comptables, normatives et notariales, dont la cote d’archive est toujours indiquée, afin de pouvoir remonter aux documents.  Finalement, des données bibliographiques peuvent être ajoutées, afin de fournir des renseignements supplémentaires sur des personnes et des fonctions."
```

Comment:
There are some double spaces here. Should those be newlines? Or just a single space?

Reply:
That is what the conversion did to the old file. I just left it how it was.